index.html

```diff
@@ -184,9 +211,11 @@ <h1>Contact me</h1>
                 <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
                     I’d love to help with it!
                 </p>
-                <form action="https://formspree.io/f/{form_id}" method="post">
+               
+                <form  action="https://formspree.io/f/{form_id}" method="post" id="formValidation">
                     <input type="text" name="name" class="lorem" placeholder="Lorem Ipsum" required></input>
-                    <input type="email" name="email" class="email" placeholder="loremipsum@gmail.com" required></input>
+                    <input type="email" name="email" class="email" placeholder="loremipsum@gmail.com"></input>
+                    <small id="smallError"></small>
                     <textarea class="textArea" name="comment" rows="8" placeholder="Write your message here..." maxlength="500" required></textarea>
                     <button type="submit" class="btnGet">Get in touch</button>     
                 </form> 
```

Comment:
- [ ] Your form validation is working properly but let me suggest you can change the placeholder for name and email just like full name and your email something like that and can check the action url it is not working when I sent the message show like below the picture.

![Screenshot (200)](https://github.com/mioOleen/Portfolio-1/assets/118582316/05b99d47-da65-4cf8-9614-674b96330696)